src/models/Artifact.ts

```diff
@@ -7,6 +9,14 @@ export const artifactTypes = [
 
 export type ArtifactType = typeof artifactTypes[number]
 
+export const artifactTypeIconMap: Record<ArtifactType | 'default', Icon> = {
+  default: 'FingerPrintIcon',
+  markdown: 'DocumentTextIcon',
+  table: 'TableIcon',
+  result: 'CalculatorIcon',
+  unknown: 'FingerPrintIcon',
+} satisfies Record<ArtifactType | 'default', Icon>
```

Comment:
So if you have satisfies you should remove the type from the const itself. Having both negates the satisfies I believe. I'd also still add the `as const`. 

```suggestion
export const artifactTypeIconMap = {
  default: 'FingerPrintIcon',
  markdown: 'DocumentTextIcon',
  table: 'TableIcon',
  result: 'CalculatorIcon',
  unknown: 'FingerPrintIcon',
} as const satisfies Record<ArtifactType | 'default', Icon>
```

Reply:
Oh TIL - i'll update that, thanks for catching that